spec/decorators/content_item_decorator_spec.rb

```diff
@@ -4,11 +4,19 @@
   describe "#last_updated" do
     let(:content_item) { build(:content_item, public_updated_at: nil).decorate }
 
-    it 'displays Never when content item has not been updated' do
+    it "displays Never when content item has not been updated" do
       expect(content_item.last_updated).to eq('Never')
     end
   end
 
+  describe "#feedex_link" do
+    let(:content_item) { build(:content_item, base_path: "/the-base-path").decorate }
+
+    it "has a link to FeedEx" do
+      expect(content_item.feedex_link).to include("/the-base-path")
```

Comment:
I would try to make this test more robust. We would need to test the text and the link.